lib/utils/android.ts

```diff
@@ -8,7 +13,7 @@ namespace Android {
     }, lazy);
 
     function getProperty(name: string): string | undefined {
-        const handle = Process.findModuleByName("libc.so")?.findExportByName("__system_property_get");
+        const handle = Module.findExportByName ? Module.findExportByName("libc.so", "__system_property_get") : Process.findModuleByName("libc.so")?.findExportByName("__system_property_get");
```

Comment:
Sorry, I don't get this change: `Process.findModuleByName()?.findExportByName("")` already existed in Frida 16

Reply:
`Process.findModuleByName("libc.so")?.findExportByName("__system_property_get")` works in both Frida 16 an 17 (?)